src/editor/EditorCommandHandlers.js

```diff
@@ -1137,6 +1148,41 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+    function _execCommand(cmd) {
+        window.document.execCommand(cmd);
+    }
+	
+    var _execCommandCut = (function() {
+        if(document.queryCommandSupported("cut")) {
+            return function() {
+                _execCommand("cut");
+            };
+        }
+        return function() {
+            Dialogs.showModalDialogUsingTemplate(Mustache.render(dialogTemplate, commandKey));
+            return ignoreCommand();
+        };
+    }());
+	
+    var _execCommandCopy = (function() {
+        if(document.queryCommandSupported("copy")) {
+            return function() {
+                _execCommand("copy");
+            };
+        }
+        return function() {
+            Dialogs.showModalDialogUsingTemplate(Mustache.render(dialogTemplate, commandKey));
+            return ignoreCommand();
+        };
+    }());
+	
+    function _execCommandPaste(){
```

Comment:
Add a comment above this indicating that we ignore the value of `document.queryCommandSupported("paste")`, because browsers don't honour it due to security concerns.